VersionControlUI/src/DiffManager.cpp

```diff
@@ -94,9 +94,11 @@ DiffSetup DiffManager::initializeDiffPrerequisites(QString oldVersion, QString n
 
 	// load newer version
 	diffSetup.newVersionManager_ = createTreeManagerFromVersion(diffSetup.repository_, diffSetup.newVersion_);
+	diffSetup.newVersionManager_->setRevisionName("new");
```

Comment:
At the moment we don't need that, no. 
But it would also work with just using the revision sha1. Should I change it and call it `revisionHash_` or similar?